src/tiled/scriptfileinfo.cpp

```diff
@@ -242,3 +242,74 @@ QString ScriptFileInfo::symLinkTarget(const QString &file) const
 {
     return QFileInfo(file).symLinkTarget();
 }
+
+QString ScriptFileInfo::cleanPath(const QString &file) const
+{
+    return QDir().cleanPath(file);
+}
+
+QString ScriptFileInfo::fromNativeSeparators(const QString &file) const
+{
+    return QDir().fromNativeSeparators(file);
+}
+
+QString ScriptFileInfo::toNativeSeparators(const QString &file) const
+{
+    return QDir().fromNativeSeparators(file);
+}
+
+bool  ScriptFileInfo::cd(const QString &file)
+{
+    return QDir().cd(file);
+}
+
+uint  ScriptFileInfo::count(const QString &file) const
+{
+    return QDir(file).count();
+}
+
+QString ScriptFileInfo::dirName(const QString &file) const
+{
+    return QDir(file).dirName();
+}
+
+bool  ScriptFileInfo::mkdir(const QString &file) const
+{
+    return QDir().mkdir(file);
+}
+
+bool  ScriptFileInfo::mkpath(const QString &file) const
+{
+    return QDir().mkpath(file);
+}
+
+bool  ScriptFileInfo::remove(const QString &file)
+{
+    return QDir().remove(file);
+}
+
+bool  ScriptFileInfo::removeRecursively(const QString &file)
+{
+    return QDir(file).removeRecursively();
+}
+
+bool  ScriptFileInfo::rename(const QString &oldName, const QString &newName)
+{
+    return QDir().rename(oldName, newName);
```

Comment:
I think here we should also just use the static [`QFile::rename`](https://doc.qt.io/qt-5/qfile.html#rename-1), which should also work for directories (though the docs don't say so).

Reply:
sounds good.